runtime/compiler/env/j9method.cpp

```diff
@@ -7530,6 +7530,7 @@ static TR::DataType typeFromSig(char sig)
       {
       case 'L':
       case '[':
+      case 'Q': //TODO: need reconsideration if value types are passed by value
```

Comment:
As above, we should track this in a new issue instead of marking this with a TODO.